src/cleed_io/cleed_io.c

```
@@ -0,0 +1,26 @@
+/*********************************************************************
+ *                       CLEED_IO.C
+ *
+ *  Stub I/O layer for CLEED curve data.
+ *********************************************************************/
+
+// cppcheck-suppress missingIncludeSystem
+#include <errno.h>
+
+#include "cleed_io.h"
+
+int cleed_io_read_curve(const char *path, cleed_curve *out_curve)
```

Cppcheck (2.19.0)

[style] The function 'cleed_io_read_curve' is never used.

(unusedFunction)